cmp/dataview/DataViewModel.ts

```diff
@@ -262,3 +265,11 @@ export class DataViewModel extends HoistModel {
         return this.gridModel.setSortBy(sorters);
     }
 }
+
+/**
+ * the `GridOptions` interface contains all `GridConfig` properties, less those
+ * specified in the above `DataViewConfig`, allowing an "escape hatch" for
+ * developers to alter the underlying `GridConfig`. Note that such alteration is
+ * not guaranteed by the DataView and may have unintended consequences.
+ */
+export interface GridOptions extends Omit<GridConfig, keyof DataViewConfig> {}
```

Comment:
nice use of typescript here!  
Not sure the `Omit<>` truly needed (its already documented so well), but why not!
 
BUT, I would probably just inline this to avoid needing to add a new identifier and doubling the documentation